src/main/drivers/memprot_stm32h7xx.c

```diff
@@ -30,7 +30,7 @@ extern uint8_t dmarwaxi_start;
 extern uint8_t dmarwaxi_end;
 
 mpuRegion_t mpuRegions[] = {
-#ifdef USE_ITCM_RAM
+#ifdef USE_ITCM
```

Comment:
Not sure about this rename - ST seems to be referring to it as 'ITCM RAM' in their literature.

Reply:
I agree that they use ITCM RAM in literature. However i also believe this name is somewhat misleading, having the _RAM suffix implies it is data memory as opposed to instructions. This could in turn prompt attempting to put data into the ITCM (when instead DTCM is made entirely for this purpose).

Im fairly unfussed though overall, as long as its consistent with CCM.